src/components/reusables/dropdown/Dropdown.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import propTypes from 'prop-types';
+
+/**
+ * @returns {object} element
+ * @param {object} props
+ */
+export default function Dropdown(props) {
+  const { active } = props;
+  return (
+    <div className={`dropdown ${active && 'active'}`}>
+      <div className="dropdown-inner">
+        <ul>
+          <li>
+            <Link to="./detail">Create</Link>
+          </li>
+        </ul>
+        <ul>
```

Comment:
#### Critical
Why not create an array of the menu items and loop through them here?